src/main/java/com/adyen/model/checkout/details/MolPayDetails.java

```diff
@@ -0,0 +1,120 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+package com.adyen.model.checkout.details;
+
+import com.adyen.model.checkout.PaymentMethodDetails;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+/**
+ * MolPayDetails
+ */
+
+public class MolPayDetails implements PaymentMethodDetails {
+    @SerializedName("issuer")
```

Comment:
Do we not need a constant here ?

Reply:
no, because this is name of the field and used only one. It is present in every model class and we will end up in tons of constants if we go for it.

Reply:
same as above.